containers/ecr-viewer/src/app/layout.tsx

```diff
@@ -28,11 +31,15 @@ export default function RootLayout({
       <html lang="en">
         <head />
         <body className="overflow-x-auto">
-          <div className="position-relative isolate">
+          <div className="minw-49rem position-relative isolate">
```

Comment:
49rem might be slightly too skinny - I'm seeing the user menu squish into the program management before the overall app stops shrinking

<img width="206" alt="image" src="https://github.com/user-attachments/assets/d57d7a9d-a72e-4018-a77b-2ac000ae4cce" />